test/memoize-one.spec.ts

```diff
@@ -0,0 +1,727 @@
+import memoize, { EqualityFn } from '../src/memoize-one';
+import isDeepEqual from 'lodash.isequal';
```

Comment:
I recommend to use `lodash/isEqual` or micro alternative like `dequal`.

Reply:
this is just for a test. is lodash/isEqual different to lodash.isequal?

Reply:
per method packages will not be supported in lodash v5. Monolithic package is recommended way.